src/navigator-card-body.jsx

```diff
@@ -222,6 +327,11 @@
         }
     };
 
+    const handleContextMenu = () => {
+        const sel = path ? path[path.length - 1] : undefined;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-237-20240201-172301-e0f83b57-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)